webapp/src/modules/translation/locales/en.json

```
@@ -866,7 +866,7 @@
       "cross_chain": "Estimated fee includes the network cost and the cross chain purchase fee that you have to pay directly with your wallet.",
       "same_network": "Estimated fee includes the network cost that you have to pay directly with your wallet."
     },
-    "insufficient_funds": "You don’t have enough funds in {token} to pay for this item. Get MANA, pay with a different token<card>, or pay by card</card>.",
+    "insufficient_funds": "You don’t have enough funds in {token} to pay for this item. Get MANA, <orPayWithOtherToken> or </orPayWithOtherToken> pay with a different token<card>, or pay by card</card>.",
```

You might get an unexpected whitespace from this translation:
```suggestion
    "insufficient_funds": "You don’t have enough funds in {token} to pay for this item. Get MANA, <orPayWithOtherToken>or </orPayWithOtherToken> pay with a different token<card>, or pay by card</card>.",
```